.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           - 13306:3306
         # Set health checks to wait until postgres has started
```

Comment:
Comment says Postgres, we're pinging MySQL, and running the tests on Postgres and SQLite?